vnext/local-cli/generate-windows.js

```diff
@@ -1,39 +0,0 @@
-'use strict';
-
-const fs = require('fs');
-const path = require('path');
-const {
-  copyProjectTemplateAndReplace,
-  installDependencies,
- } = require('./generator-windows');
-
-/**
- * Simple utility for running the Windows generator.
- *
- * @param  {String} projectDir root project directory (i.e. contains index.js)
- * @param  {String} name       name of the root JS module for this app
- * @param  {String} ns         namespace for the project
- * @param  {Object} options    command line options container
- */
-function generateWindows (projectDir, name, ns, options) {
```

Comment:
This and a couple of other files weren't getting linted for some reason. After linting Git can't connect the dots and realize there was a rename/edit.

You can look at the first commit to see the exact changes before running `yarn lint:fix`